README.md

```diff
@@ -210,6 +212,45 @@ If you do want to sign JavaScript, please enable it with the `signJavaScript`
 parameter. As far as we are aware, there are no benefits to signing
 JavaScript files, so we do not by default.
 
+# Use with Cloud HSM Providers
+
+Since 2023, Microsoft requires that Windows software be signed with an extended validation (EV) certificate in order to avoid a Windows Defender popup. This presents a challenge for developers who are used to building and signing in the cloud or some continuous integration setup like GitHub Actions, CircleCI, Travis CI, or AppVeyor because EV certificates require a Hardware Security Module (HSM). Or, in simpler words: If you want to code sign your Windows software so that your customers don't see a scary Windows Defender dialog, you need to code sign on a computer with a USB device plugged in. 
```

Comment:
I was reading through the [DigiCert press release for this change](https://www.digicert.com/blog/improved-requirements-for-code-signing-key-protection) and I guess there are two nits to disambiguate here:

* The new standards were released by the [CA/Browser Forum](https://cabforum.org/2022/04/06/ballot-csc-13-update-to-subscriber-key-protection-requirements/) and not just Microsoft.
* Do OV certificates still exist in 2024? What I saw from the DigiCert blog, OV certificates could still be issued but the June 1, 2023 change specifically meant that OV certs required EV-level key security (i.e. HSMs).

Reply:
That's correct.

- EV was always needed to fully prevent the windows defender popup from the beginning (OV certs would hide the popup only after Microsoft Defender "learned" about the executable)
- Since 2023 OV certificates now also require HSMs/ cloud key solutions (and got more expensive) so if you're jumping through the hoops for HSM/cloud key you might as well get an EV anyway.